functionary/ui/templates/core/function_detail.html

```diff
@@ -69,6 +57,34 @@ <h1 class="title is-1">
                 <span class="ml-4">{{ function.output_format }}</span>
             </div>
         {% endif %}
+        <label class="label ml-4">Create Task:</label>
+        <div class="has-background-white-ter ml-4">
+            <form id="djangoForm"
+                    method="post"
+                    action="{% url 'ui:function-execute' %}">
+                {% csrf_token %}
+                <input type="hidden" name="function_id" value="{{ function.id }}"/>
+                <div class="column has-addons">
+                    {{ form.non_field_errors }}
+                    {{ form }}
+                    {% if missing_variables %}
+                        <div class="field ml-4 missing-variables has-background-danger-light has-text-danger">
+                            <span>Tasking this function may not work as expected, the following variables are missing:</span>
```

Comment:
Super nitpick: I think the comma needs to be a semicolon for this to read correctly.

Reply:
Done